third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -2812,7 +2812,8 @@ std::unique_ptr<media::Renderer> WebMediaPlayerImpl::CreateRenderer(
   // `media_task_runner_` is always true, use an if statement to avoid
   // potential build warning on unreachable code.
   if (media_task_runner_) {
-    return std::make_unique<media::StarboardRenderer>(media_task_runner_);
+    return std::make_unique<media::StarboardRenderer>(media_task_runner_,
```

Comment:
For absolute cleanliness, this should _also_ have an `#if COBALT` switch here.

I know it's a tiny switch, but still. Also is this small change upstreamable ?

Reply:
We have a TODO b/375278384, which will make the code change upstreamable.

This small change is from https://github.com/youtube/cobalt/pull/4283, not the original Chromium's code.